lms/static/scripts/frontend_apps/components/AppRoot.tsx

```diff
@@ -15,33 +17,105 @@ export type AppRootProps = {
   services: ServiceMap;
 };
 
+/**
+ * Return dummy configuration for the file picker app. This will be replaced
+ * with a call to the endpoint created in https://github.com/hypothesis/lms/pull/5120.
+ */
+/* istanbul ignore next */
+async function loadDummyFilePickerConfig(
+  config: ConfigObject
+): Promise<ConfigObject> {
```

Comment:
See last commit(s) in https://github.com/hypothesis/lms/pull/5150 for the real version of this function.